tests/run-unit-tests.scm

```diff
@@ -267,6 +286,8 @@
   (if (= nb-good nb-total)
       (begin
         (print "PASSED ALL " nb-total " UNIT TESTS\n")
+        (if show-coverage?
```

Comment:
Currently this prints a list of symbols... Shouldn't there be a header to indicate what this list is?